noxfile.py

```diff
@@ -0,0 +1,152 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2020 Google LLC
+#
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file or at
+# https://developers.google.com/open-source/licenses/bsd
+
+
+from __future__ import absolute_import
+
+import nox
+import os
+# import shutil
```

Comment:
Even if these will be added later, better to just remove it now than leave it in history but commented.

```suggestion
```

Reply:
Agreed. Done.